docs/config.md

```
@@ -18,6 +18,7 @@
    - [3.9 DB バックアップ設定](#39-db-バックアップ設定dbbackup)
    - [3.10 Web Fetch 設定](#310-web-fetch-設定web_fetch)
    - [3.11 エージェント定義](#311-エージェント定義agentsid)
+   - [3.12 Webhook 設定](#312-webhook-設定webhooks)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)